src/wallet/ismine.h

```diff
@@ -14,7 +14,27 @@
 class CWallet;
 class CScript;
 
-/** IsMine() return codes */
+/**
+ * IsMine() return codes.
+ * 
+ * ISMINE_NO: the scriptPubKey is not in the wallet.
+ * Other types depend on ScriptPubKeyMan implementation. For DescriptorScriptPubKeyMan
+ * and future ScriptPubKeyMan, only `ISMINE_NO` and `ISMINE_SPENDABLE` will be used.
+ * 
+ * For LegacyScriptPubKeyMan,
+ * ISMINE_NO: the scriptPubKey is not in the wallet;
+ * ISMINE_WATCH_ONLY: the scriptPubKey has been imported into the wallet;
+ * ISMINE_SPENDABLE: the scriptPubKey corresponds to an address owned by the wallet user (can spend with the private key);
+ * ISMINE_USED: the scriptPubKey corresponds to a used address owned by the wallet user;
```

Comment:
`ISMINE_USED` is used by all `ScriptPubKeyMan` types, not just legacy.

Reply:
Thx! Anything else I need to address?